modules/oe_content_persistent/src/Form/PurlSettingsForm.php

```diff
@@ -41,6 +74,13 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#description' => $this->t('The base URL to use for building persistent URLs'),
     ];
 
+    $form['supported_entity_types'] = [
+      '#type' => 'checkboxes',
+      '#options' => $this->getEntityTypeOptions(),
+      '#default_value' => $config->get('supported_entity_types'),
+      '#title' => $this->t('Supported entity types'),
```

Comment:
What happens if none are the entity types are supported? How does the whole thing behave? Can you check? because we might need to make this required for at least 1.

Reply:
Indeed we will trigger some exceptions. I made the field required.